DTS-STN/Alpha-Site
# Projects redesign

# Description

[Projects page redesign](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-671)

Update the content and layout for the projects page to match the re-design on [Figma](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/SARAK-BDM-DECD-SC-Labs?node-id=9561%3A25275). Page content now using data from AEM. My previous pr for the card component had some styling issues, fixed in this pr. French content are not available at the moment, will update again once the French content get approved.

<img width="386" alt="Screen Shot 2022-06-28 at 1 10 08 PM" src="https://user-images.githubusercontent.com/64853947/176242309-b1adfd42-06cc-4e6b-8544-b9ddbe15fb36.png">

Note: Regarding the disclaimer banner on the top, it has it's own ticket on jira. I'm not sure if it is ready for the dev, will ask Celeste. 

## Test Instructions

1. yarn dev
2. go to projects page
3. make sure it matches the design on Figma

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)
